Jenkinsfile

```diff
@@ -180,78 +163,70 @@ pipeline {
+                                echo "Coverity is only run on master"
+                            } else {
+                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
+                                    withCredentials([string(credentialsId: 'alliance-coverity-token', variable: 'COVERITY_TOKEN')]) {
+                                        withEnv(["PATH=${tool 'coverity-linux'}/bin:${env.PATH}"]) {
+                                            configFileProvider([configFile(fileId: 'coverity-maven-settings', replaceTokens: true, variable: 'MAVEN_SETTINGS')]) {
+                                                sh 'cov-build --dir cov-int mvn -DskipTests=true -DskipStatic=true install -B -pl !$DOCS --settings $MAVEN_SETTINGS'
+                                                sh 'tar czvf alliance.tgz cov-int'
+                                                sh 'curl --form token=$COVERITY_TOKEN --form email=cmp-security-team@connexta.com --form file=@alliance.tgz --form version="master" --form description="Description: Alliance CI Build" https://scan.coverity.com/builds?project=codice%2Falliance'
                                             }
                                         }
                                     }
                                 }
                             }
                         }
-                    },
-                    codecov: {
-                        node('linux-large') {
-                            script {
-                                retry(3) {
-                                    checkout scm
-                                }
-                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
-                                    withCredentials([string(credentialsId: 'DDF_CodeCov', variable: 'CODECOV_TOKEN')]) {
-                                        sh 'mvn clean install -B -T 1C -pl !$ITESTS'
-                                        sh 'curl -s https://codecov.io/bash | bash -s - -t ${CODECOV_TOKEN}'
-                                    }
-                                }
+                    }
+                }
+                stage ('Codecov') {
+                    agent { label 'linux-small' }
+                    steps {
+                        retry(3) {
+                            checkout scm
+                        }
+                        withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
+                            withCredentials([string(credentialsId: 'DDF_CodeCov', variable: 'CODECOV_TOKEN')]) {
```

Comment:
@mcalcote I realize this was in use prior as well, but are these the correct credentials for Alliance?

Reply:
@shaundmorris helped me update this with Alliance specific credentials